src/protocols/ssh/ssh.c

```diff
@@ -340,6 +346,12 @@ void* ssh_client_thread(void* data) {
             break;
         }
 
+        /* Make sure ssh_client_thread can be terminated anyway */
```

Comment:
I'm not sure what you mean here by "make sure ssh_client_thread can be terminated anyway".